docs/index.bs

```diff
@@ -66,6 +66,10 @@ spec: page-visibility; urlPrefix: https://www.w3.org/TR/page-visibility/
     type: enum; text: VisibilityState; url: VisibilityState
     type: attribute; text: visibilityState; for: Document; url: dom-document-visibilitystate
 
+spec: page-lifecycle; urlPrefix: https://wicg.github.io/page-lifecycle/spec.html
+    type: dfn; text: frozen; url: frozen
+    type: dfn; text: owning document; url: dedicatedworkerglobalscope-owning-document
```

Comment:
@tabatkins can we get https://wicg.github.io/page-lifecycle/spec.html added to bikeshed's database?

Reply:
Should be done, give it an hour or so.